app/cdap/text/text-en.yaml

```diff
@@ -3432,6 +3432,53 @@ features:
       success: You have successfully created the pipeline "{pipelineName}".
     Skip: Skipped
   WranglerNewUI:
+    GridPage:
+      addTransformationPanel:
+        applyStep: Apply Step
+        columnsSelected: columns selected
+        columns: Columns
+        dataQuality: Data Quality
+        done: Done
+        nullValues: Null Values
+        selectColumnPara: Select column(s) to apply this function
```

Comment:
Can this be more concise? It fills the entire space currently and doesn't read right. Also, can it be aware of whether the directive is multi-column?

Reply:
Fixed the issue.
Select Column -  for single Column Selection
Select Columns - for multiple Column Selection.